### DISCUSSION TITLE
2010/11/08/%E7%A1%AC%E7%9B%98%E7%9A%84%E9%80%9F%E5%BA%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#301] [Announcements]
# 2010/11/08/%E7%A1%AC%E7%9B%98%E7%9A%84%E9%80%9F%E5%BA%A6

以前一直没有考虑过，不同硬盘的读写速率是有很大区别的。因为硬盘上标注的参数一般就只有容量和转速，所以我一直以为转速是影响硬盘速度的唯一指标。最近才发现，其实不是这样的：

https://blog.qizhen.xyz/2010/11/08/%E7%A1%AC%E7%9B%98%E7%9A%84%E9%80%9F%E5%BA%A6

**Comment by ruanqizhen** [comment]
ruanqizhen
2010年11月11日 20:31 编辑
我后来又意识到，在磁盘边缘读取数据比在磁盘中心处读取数据块。所以，也许系统盘应该画在磁盘外部区域才好。

回复
